DigitalLearningSolutions.Data/DataServices/ActivityDataService.cs

```diff
@@ -20,20 +19,35 @@ public ActivityDataService(IDbConnection connection)
             this.connection = connection;
         }
 
-        public IEnumerable<MonthOfActivity> GetActivityInRangeByMonth(int centreId, DateTime start, DateTime end)
+        public IEnumerable<ActivityLog> GetRawActivity(int centreId, ActivityFilterData filterData)
         {
-            return connection.Query<MonthOfActivity>(
+            return connection.Query<ActivityLog>(
                 @"SELECT
-                        LogYear AS Year,
-                        LogMonth AS Month,
-                        SUM(CONVERT(INT, Completed)) AS Completions,
-                        SUM(CONVERT(INT, Evaluated)) AS Evaluations,
-                        SUM(CONVERT(INT, Registered)) AS Registrations 
+                        LogDate,
+                        LogYear,
+                        LogQuarter,
+                        LogMonth,
+                        Registered,
+                        Completed,
+                        Evaluated
                     FROM tActivityLog
-                        WHERE (LogDate > @start AND LogDate < @end AND CentreID = @centreId)
-                    GROUP BY LogYear, LogMonth
-                    ORDER BY LogYear, LogMonth",
-                new { centreId, start, end }
+                        WHERE (LogDate > @startDate
+                            AND LogDate < @endDate

```

Comment:
'LogDate' is a full datetime with a millisecond timestamp rather than just a date, so in practice it will be included unless it occured at exactly midnight. However it would still be more correct to change it.